examples/macro.rs

```diff
@@ -0,0 +1,13 @@
+#[macro_use] extern crate bidir_map;
+
+fn main() {
+	let map = bidir_map! {
+		42 => "foo",
+		64 => "bar",
+	};
+	assert_eq!(map.get_by_first(&42), Some(&"foo"));
+	assert_eq!(map.get_by_first(&64), Some(&"bar"));
+	assert_eq!(map.get_by_second(&"foo"), Some(&42));
+	assert_eq!(map.get_by_second(&"bar"), Some(&64));
+	println!("{:?}", map);
+}
```

Comment:
Not sure this is necessary, given the fact that perfectly identical code is in doctests?

Reply:
Yea, it's not strictly "necessary" from a redundancy standpoint, but I think it's still useful to show the usage of the retrieval methods in the example so that someone who looks at the example before looking at the doc already gets all the info they need for the common case :)

Reply:
But the examples are in the doc though? I don't really get the point here 🤔

Reply:
Right, so should I remove the example?

Reply:
That would be great, thank you

Reply:
Done :)